### PR TITLE
chore: Update deps

### PR DESCRIPTION
**Pull request by nicholaspai**

### CODE REVIEW

File: src/contracts/hubPool.ts [hunk 1]
```diff
@@ -2,14 +2,15 @@ import { HubPool, HubPool__factory, HubPoolInterface } from "@across-protocol/co
...
+import { Signer } from "ethers";
 const { Balances } = uma.utils;
 
 export type Instance = HubPool;
 export const Factory = HubPool__factory;
 export type Interface = HubPoolInterface;
 
 export function connect(address: string, provider: SignerOrProvider): Instance {
-  return Factory.connect(address, provider);
+  return Factory.connect(address, provider as unknown as Signer);
```
Comment (nicholaspai) [inline src/contracts/hubPool.ts:13]: Necessary because `SignerOrProvider` from `uma/sdk` is on an older `ethers`

File: yarn.lock [hunk 2]
```diff
@@ -1097,24 +1107,32 @@
...
-    ethers "^5.5.4"
 
-"@eth-optimism/core-utils@0.8.4":
-  version "0.8.4"
-  resolved "https://registry.yarnpkg.com/@eth-optimism/core-utils/-/core-utils-0.8.4.tgz#617883bf85c438e40e0e09de427582390359dff9"
-  integrity sha512-HayyAM8rawvRY8wg/Iy/5O4Gn6VPYxlgaQ1vLcbwDDMvXOCbIjONbOBqi+v7LKVPuvfL6xY+3aMibm5tWtrn/g==
+"@eth-optimism/core-utils@0.8.1":
+  version "0.8.1"
+  resolved "https://registry.yarnpkg.com/@eth-optimism/core-utils/-/core-utils-0.8.1.tgz#ee7d8815f9ddb67ca3e2f3ec1d8515a46af5caf6"
+  integrity sha512-GJJvw9cBekvR1xH/f2jpgp6MnbWlMBaVm1h3BA0sEzmlaFufTG4ybGrHkL3nQT0Vqtjs3LYjaExkVmsUzpb7xg==
```
Comment (pxrl) [inline yarn.lock:1135]: This is interesting - are we downgrading here?

File: yarn.lock [hunk 3]
```diff
@@ -2865,6 +2925,204 @@
...
+    "@nomicfoundation/solidity-analyzer-darwin-arm64" "0.1.0"
+    "@nomicfoundation/solidity-analyzer-darwin-x64" "0.1.0"
+    "@nomicfoundation/solidity-analyzer-freebsd-x64" "0.1.0"
+    "@nomicfoundation/solidity-analyzer-linux-arm64-gnu" "0.1.0"
+    "@nomicfoundation/solidity-analyzer-linux-arm64-musl" "0.1.0"
+    "@nomicfoundation/solidity-analyzer-linux-x64-gnu" "0.1.0"
+    "@nomicfoundation/solidity-analyzer-linux-x64-musl" "0.1.0"
+    "@nomicfoundation/solidity-analyzer-win32-arm64-msvc" "0.1.0"
+    "@nomicfoundation/solidity-analyzer-win32-ia32-msvc" "0.1.0"
+    "@nomicfoundation/solidity-analyzer-win32-x64-msvc" "0.1.0"
```
Comment (pxrl) [inline yarn.lock:3124]: Damn..would be so nice if we had a way of excluding these.

File: yarn.lock [hunk 4]
```diff
@@ -8326,7 +8669,7 @@ ethereum-common@^0.0.18:
   resolved "https://registry.yarnpkg.com/ethereum-common/-/ethereum-common-0.0.18.tgz#2fdc3576f232903358976eb39da783213ff9523f"
   integrity sha1-L9w1dvIykDNYl26znaeDIT/5Uj8=
 
-ethereum-cryptography@^0.1.2, ethereum-cryptography@^0.1.3:
+ethereum-cryptography@0.1.3, ethereum-cryptography@^0.1.2, ethereum-cryptography@^0.1.3:
```
Comment (pxrl) [inline yarn.lock:8672]: Weird double-up on ethereum-cryptography@0.1.3 here. One is pegged to the exact 0.1.3, the other is pegged to the minor.

Probably nothing to do - just a shame the tools do this :(

**Review by pxrl — APPROVED** [review]
Nice to get up to 5.7.x - this will enable some future gas price estimation improvements.

I see there are some GitHub lint observations at the bottom; this looks new to me, and seems unrelated to the PR, so I've ignored it - but we might want to take a look separately.

File: yarn.lock (hunk 2 above)
Comment (nicholaspai) [reply yarn.lock]: Interesting, it looks like this is required by an older version of eth-optimism/contracts. I'm not sure yarn.lock does a good job of deleting older deps so I manually deleted this line (and the older eth-optimism/contracts) and it then deleted a bunch of older files

File: yarn.lock (hunk 4 above)
Comment (nicholaspai) [reply yarn.lock:8432]: Yeah I just tried cleaning up the yarn.lock with this tool: https://www.npmjs.com/package/yarn-deduplicate

Seems useful and added a bunch of edits in this commit https://github.com/across-protocol/sdk-v2/pull/117/commits/7949b87c429c8b0333d7f7740d545a68c1c1997c

Comment (nicholaspai) [reply yarn.lock:8432]: This tool is also very useful https://www.npmjs.com/package/yarn-check

**Pull request merged**